UI/MainForm.cs

```diff
@@ -1247,6 +1247,56 @@ private void _checkForOrphanedJoysticks(bool showNotNecessaryMessage)
             }
         }
 
+        private void _checkForOrphanedMidiBoards(bool showNotNecessaryMessage)
+        {
+            List<string> serials = new List<string>();
+            List<string> NotConnectedMidiBoards = new List<string>();
+
+            foreach (MidiBoard mb in execManager.GetMidiBoardManager().GetMidiBoards())
+            {
+                serials.Add($"{mb.Name} {SerialNumber.SerialSeparator}{mb.Serial}");        
+            }
+
+            if (configFile == null) return;
+
+            foreach (OutputConfigItem item in configFile.GetOutputConfigItems())
+            {
+                if (item.DisplaySerial.Contains(MidiBoard.SerialPrefix) &&
+                    !serials.Contains(item.DisplaySerial) &&
+                    !NotConnectedMidiBoards.Contains(item.DisplaySerial))
+                {
+                    NotConnectedMidiBoards.Add(item.DisplaySerial);
+                }
+            }
+
+            foreach (InputConfigItem item in configFile.GetInputConfigItems())
+            {
+                if (item.ModuleSerial.Contains(MidiBoard.SerialPrefix) &&
+                    !serials.Contains(item.ModuleSerial) &&
+                    !NotConnectedMidiBoards.Contains(item.ModuleSerial))
+                {
+                    NotConnectedMidiBoards.Add(item.ModuleSerial);
+                }
+            }
+
+            if (NotConnectedMidiBoards.Count > 0)
```

Comment:
This can be unwound, changed to `if (NotConnectedMidiBoards.Count == 0) return`;

Reply:
Same style discussion. I need to think about in which cases I like something like that and when not.